Packet++/header/IgmpLayer.h

```diff
@@ -283,15 +285,17 @@ class IgmpV2Layer : public IgmpLayer
 	 * @param[in] prevLayer A pointer to the previous layer
 	 * @param[in] packet A pointer to the Packet instance where layer will be stored in
 	 */
-	IgmpV2Layer(uint8_t* data, size_t dataLen, Layer* prevLayer, Packet* packet);
+	IgmpV2Layer(uint8_t* data, size_t dataLen, Layer* prevLayer, Packet* packet)
+		: IgmpLayer(data, dataLen, prevLayer, packet, IGMPv2) {}
 
 	/**
 	 * A constructor that allocates a new IGMPv2 header
 	 * @param[in] type The message type to set
-	 * @param[in] groupAddr The multicast address to set. This is an optional parameter and has a default value of IPv4Address#Zero
+	 * @param[in] groupAddr The multicast address to set. This is an optional parameter and has a default value of unspecified IPv4 address

```

Comment:
Can you add `unspecified/zero` instead of just `unspecified`?

Reply:
Done